week3/homework/src/index.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+
+const { readTodo, clearTodos, markAsDone, markAsNotDone } = require('./actions');
+
+const Todo = require('./todo');
+const FILENAME = `${__dirname}/todoList.json`;
+const todo = new Todo(FILENAME);
+
+app.use(bodyParser.json());
+
+const TODO_SLUG = 'todos';
+// Get a single to-do with ID :id
+app.get(`/${TODO_SLUG}/:id`, readTodo.bind(null, todo));
+
+// Clear the list of to-dos
+app.delete(`/${TODO_SLUG}`, clearTodos.bind(null, todo));
+
+// Set the done flag of a single to-do to true
+app.post(`/${TODO_SLUG}/:id/done`, markAsDone.bind(null, todo));
```

Comment:
Thanks, it can be ```put``` of course, I've tried it and it works also, but in the homework the question was to do it this way: markAsDone (POST /todos/:id/done).